### PR TITLE
Add normalization scripts

### PR DESCRIPTION
**Pull request by loginaway**

### CODE REVIEW

File: examples/postprocess_aser/aser_filter.ipynb [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+      "\u001b[0;31mRuntimeError\u001b[0m: Expected all tensors to be on the same device, but found at least two devices, cpu and cuda:0! (when checking argument for argument index in method wrapper__index_select)"
```
Comment (seanliu96) [inline examples/postprocess_aser/aser_filter.ipynb]: delete this one?

Comment (loginaway) [reply examples/postprocess_aser/aser_filter.ipynb]: This file can be removed. Another temporary file.

File: examples/postprocess_aser/normalization.py [hunk 2]
```diff
@@ -0,0 +1,607 @@
...
+        last_j = 0
+        for i, j in spans:
+            norm_node_wrds.extend(node_wrds[last_j:i])
+            p2indices[span2pers[(i,j)]].append(len(norm_node_wrds))
+            norm_node_wrds.append(replace_dict[(i,j)])
+            last_j = j
+        norm_node_wrds.extend(node_wrds[last_j:])
+        return {'norm_node': ' '.join(norm_node_wrds), 'p2i': dict(p2indices)}
+
+
```
Comment (seanliu96) [inline examples/postprocess_aser/normalization.py:591]: remove redundant empty lines (no more than two)

File: examples/postprocess_aser/normalization.py [hunk 3]
```diff
@@ -0,0 +1,607 @@
...
+        for info in info_list:
+            info = eval(info)
+
+            data = self.process_info(info)
+
+            print(' '.join(info['words']))
+            if data:
+                subj = ' '.join(data['subj'][0][2])
+                objs = [' '.join(i[2]) for i in data['objs']]
+                print('【SUBJ】:{}\t【OBJS】: {}'.format(subj, ', '.join(objs)))
```
Comment (seanliu96) [inline examples/postprocess_aser/normalization.py:607]: 【 -> [
】-> ]

**Pull request merged**